shared/locales/activities/activities.en-US.properties

```diff
@@ -1,31 +1,50 @@
 # Default activities label
 
-# PICK
-default-activity-pickimage    = Take picture
+# LOCALIZATION NOTE This strings are used to describe actions available for
+#   which an application can be associated for default launch.
+#   e.g. "Open link" -> Firefox
+#        "Open image"-> Gallery
+#
+# Some separated actions are grouped as one for this purposes, so when an app
```

Comment:
I'm not sure that the information from this point on is really useful to localizers. I think "developer-relevant" comments should be in the code, not in localization files.